### PR TITLE
Generate temporary directory name in HDFS

### PR DESCRIPTION
**Pull request by wendigo**
temporaryStagingDirectory is a directory in the HDFS so using locally generated tmp directory path
is incorrect on MacOS (as java.io.tmpdir points to /var/folders/ss/.../T which doesn't exist on HDFS).

### CODE REVIEW

**Comment by wendigo** [comment]
cc @homar

**Review by homar — APPROVED** [review]

**Review by hashhar — COMMENTED** [review]
Looks good. But the commit makes it look like the change is in the Hive connector itself.

Maybe something like "Use /tmp for temporaryStagingDirecotry in Hive tests" with the body being the same.

**Review by losipiuk — APPROVED** [review]

File: plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java [hunk 1]
```diff
@@ -652,7 +652,7 @@ public void setupClass()
         executor = newCachedThreadPool(daemonThreadsNamed("hive-%s"));
         heartbeatService = newScheduledThreadPool(1);
         // Use separate staging directory for each test class to prevent intermittent failures coming from test parallelism
-        temporaryStagingDirectory = createTempDirectory("trino-staging-");
+        temporaryStagingDirectory = Paths.get("/tmp/" + UUID.randomUUID());
```
Comment (losipiuk) [inline plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java:655]: Looks fine - unless this sometimes maps to local file system. Then it is suboptimal. Maybe the path should be provided from the `run_hive_*_test.sh` script?

Comment (findepi) [reply plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java:655]: add a commentm why, so it;s not tempting to "fix" this code in the future

**Comment by mosabua** [comment]
:wave: @wendigo - this PR has become inactive. Did someone just forget to merge since it does have approval from @losipiuk ?

We're working on closing out old and inactive PRs, so if you're too busy or this has too many merge conflicts to be worth picking back up, we'll be making another pass to close it out in a few weeks.

**Comment by colebow** [comment]
Looks like the CI failures would probably need to be addressed. I'm gonna close this one out due to inactivity, but please reopen if you would like to pick this back up, it seems pretty lightweight and easy to get across the finish line if we're still interested in it.

**Pull request closed without merging**